pkg/pillar/cmd/downloader/space.go

```diff
@@ -26,6 +26,9 @@ func initSpace(ctx *downloaderContext, kb uint64) {
 // Returns true if there was space
 func tryReserveSpace(ctx *downloaderContext, status *types.DownloaderStatus,
 	kb uint64) bool {
+	if status.ReservedSpace != 0 {
+		return true
```

Comment:
A debug statement would be useful here..

Reply:
Have added.

Reply:
Is this an error or merely an indication that we have already performed the reservation and shouldn't do it twice?

Reply:
I think, we can a put an error here, indicating some thing wrong with the code.